src/OpenTelemetry.Api.ProviderBuilderExtensions/OpenTelemetry.Api.ProviderBuilderExtensions.csproj

```diff
@@ -8,10 +8,11 @@
     <AnalysisLevel>latest-all</AnalysisLevel>
   </PropertyGroup>
 
-  <!--Do not run ApiCompat as this package has never released a stable version.
+  <!--Do not run ApiCompat nor Package Baseline Validation as this package has never released a stable version.
   Remove this property once we have released a stable version.-->
   <PropertyGroup Condition="'$(TargetFramework)' == 'net6.0'">
     <RunApiCompat>false</RunApiCompat>
```

Comment:
One more thing: You are mixing the old APICompat with the new Package Validation which isn't really supported. Unfortunately, the `RunApiCompat` property still exists in the new Package Validation (we should delete in our code - I'll file an issue for that). If you keep that set, you might see unexpected behavior in the new Package Validation tooling.

That goes back to my initial comment in this thread about better not using both the old and the new APICompat (PackageValidation) together.

Reply:
Are you saying we cannot keep both for the time being without them interfering with each other?

Reply:
You can keep the property as long as it is only set in an inner build (meaning only in multi-targeting projects conditioned on a TFM).

Reply:
After checking this with @utpilla , we decided to remove the old APICompat to avoid any unexpected behavior. Could you please have a look at the latest iteration?